test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/ContractAutoExpirySpecs.java

```diff
@@ -86,9 +88,243 @@ public List<HapiSpec> getSpecsInSuite() {
+                                .entityMemo("")
+                                .bytecode(initcode)
+                                .autoRenewSecs(minimalLifetime)
+                                .autoRenewAccountId(autoRenewAccount)
+                                .balance(0L)
+                                .via(creation),
+                        withOpContext(
+                                (spec, opLog) -> {
+                                    final var lookup = getTxnRecord(creation);
+                                    allRunFor(spec, lookup);
+
+                                    final var record = lookup.getResponseRecord();
+                                    final var birth = record.getConsensusTimestamp().getSeconds();
+                                    currentExpiry.set(birth + minimalLifetime);
+                                    expectedExpiryPostRenew.set(
+                                            birth
+                                                    + minimalLifetime
+                                                    + minimalLifetime
+                                                    + minimalLifetime);
+                                    opLog.info(
+                                            "Expecting post-renewal expiry of {}, current expiry"
+                                                    + " {}",
+                                            expectedExpiryPostRenew.get(),
+                                            currentExpiry.get());
+                                    final var info =
+                                            getContractInfo(contractToRenew)
+                                                    .has(
+                                                            contractWith()
+                                                                    .isNotDeleted()
+                                                                    .approxExpiry(
+                                                                            currentExpiry.get(), 1))
+                                                    .logged();
+
+                                    allRunFor(spec, info);
+                                }),
+                        sleepFor(minimalLifetime * 1_000L + 500L))
+                .when(cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)))
+                .then(
+                        /*
+                         * Funding for auto-renew account with enough balance. So when
+                         * contract successfully auto-renews on next trigger
+                         */
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, autoRenewAccount, 100 * ONE_HBAR)),
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)),
+                        getAccountBalance(autoRenewAccount).logged(),
+                        sleepFor(minimalLifetime * 1_000L + 500L),
+                        assertionsHold(
```

Comment:
More for my knowledge than anything else, what's the difference between 
```
sleepFor(minimalLifetime * 1_000L + 500L)
assertionsHold(
    // assertion 1
    // assertion 2
),
```
and
```
sleepFor(minimalLifetime * 1_000L + 500L),
// assertion 1
// assertion 2
```
?

Reply:
No difference 🙂